reqs/models.py

```diff
@@ -69,6 +69,7 @@ class Meta:
     sunset = models.DateField(blank=True, null=True)
     policy_status = models.CharField(max_length=256, blank=True)
     document_source = models.FileField(blank=True)
+    nonpublic = models.NullBooleanField()
```

Comment:
I was thinking of the (future, maybe) Requirement case (where null would effectively mean “inherit”) instead of the Policy case when I made it three-value logic; I've made it default to `False`.